src/embedded.js

```diff
@@ -219,17 +220,20 @@ class Embedded {
     }
   }
 
-  handler (alpheiosEvent, domEvent) {
+  async handler (alpheiosEvent, domEvent) {
     if (this.triggerPreCallback(domEvent)) {
       let htmlSelector = new HTMLSelector(alpheiosEvent, this.options.items.preferredLanguage.currentValue)
       let textSelector = htmlSelector.createTextSelector()
 
       if (!textSelector.isEmpty()) {
+        await this.options.load()
```

Comment:
why do we need to load them again here?

Reply:
Similiar to the previous.
I don't exactly know when, I think after some refactoring components phase, options are not been uploaded from LocalStorage, couldn't be changed after changing on Options tab.

So as I needed to work with enabling/disabling word usage examples properties in Options, I have made two hacky solutions. 
I have added 
`await this.options.load()`
in two places 
   - in `activate` - so it uploads current properties from LocalStorage
   - before making LexicalQuery - it uploads again to avoid loosing changes made through Options tab

This solution is of course temporary and is useful only to use options feature for the embed-lib.
As we talked on the meeting - Kirill is working on it in components.

Reply:
ok, I think this will need to be changed with Kirill's merge, but it is okay for the moment to allow us to further test and develop this feature.

Reply:
I've updated a UI controller so we would be able to use (and tweak) UI controller's options from within an embedded lib. This will eliminate a need of an embedded lib to have its own options objects completely. I believe it's a better way to handle this: it makes more sense for an embedded lib to create its own customized version of a UI controller and use it rather than duplicate pieces of UI controller's configuration internally.